keyboards/system76/launch_1/keymaps/levi/keymap.c

```diff
@@ -109,5 +114,22 @@ ________________________________________________________________________________
 };
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case KC_LCTL:
+        lctl_pressed = record->event.pressed;
+        break;
+    case KC_RCTL:
+        rctl_pressed = record->event.pressed;
+        break;
+    case KC_ESC:
+        esc_pressed = record ->event.pressed;
+        break;
+    };
     return true;
 }
+
+void matrix_scan_user(void) {
+    if (lctl_pressed && rctl_pressed && esc_pressed) {
+        system76_ec_unlock();
```

Comment:
Your commit that added this line should probably be squashed, but that is not a big deal.

Reply:
done!